src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/BeslutteVedtak.kt

```diff
@@ -154,7 +154,15 @@ class BeslutteVedtak(
 
         if (nesteSteg == StegType.IVERKSETT_MOT_OPPDRAG) {
             val erInnvilgetSøknadUtenUtebtalingsperioderGrunnetEndringsperioder =
-                beregningService.erAlleUtbetalingsperioderPåNullKronerIDenneOgForrigeBehandling(behandling = behandling)
+                if (featureToggleService.isEnabled(FeatureToggleConfig.BRUK_ATY_FOR_Å_AVGJØRE_DROPPE_SIMULERING)) {
+                    beregningService.erAlleUtbetalingsperioderPåNullKronerIDenneOgForrigeBehandling(
+                        behandling = behandling
+                    )
+                } else {
+                    beregningService.innvilgetSøknadUtenUtbetalingsperioderGrunnetEndringsPerioder(
+                        behandling = behandling
+                    )
+                }
 
             if (erInnvilgetSøknadUtenUtebtalingsperioderGrunnetEndringsperioder) {
```

Comment:
```suggestion
            if (erInnvilgetSøknadUtenUtebtalingsperioderGrunnetEndringsperioder && behandling.erBehandlingMedVedtaksbrevutsending()) {
```
Kan være greit å ha med